meson.build

```diff
@@ -26,7 +26,7 @@ if cc.get_id() == 'gcc'
     error('SciPy requires GCC >= 8.0')
   endif
 elif cc.get_id() == 'msvc'
-  if not cc.version().version_compare('>=1920')
+  if not cc.version().version_compare('>=19.20')
```

Comment:
this PR has a backport label but the addition of the code it changes is in a PR that is not backport--I'm tentatively removing the backport label here at this time.. ping me here and in the other PR if we really need it

Reply:
Hmm, indeed, this PR fixes e9f799acea82177af239594175469561344d2d22 which purely added error messages. So this should not need any backporting.